scripts/install.sh

```diff
@@ -20,40 +20,43 @@ help()
    echo -e "\nUsage: ENV_VAR(S) $0"\
            "--config-dir <path>"\
            "--network <network>"\
-           "--contracts-only"
+           "--skip-deployment <boolean>"\
+           "--skip-client-build <boolean>"
    echo -e "\nEnvironment variables:\n"
    echo -e "\tKEEP_ETHEREUM_PASSWORD: The password to unlock local Ethereum accounts to set up delegations."\
            "Required only for 'local' network. Default value is 'password'"
    echo -e "\tCONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Ethereum. Required for non-local network only"
```

Comment:
We no longer use the `local` network name. Also, I think the variable is also not needed for the `hardhat` network.
Maybe
```suggestion
   echo -e "\tCONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Ethereum. Required for non-development and non-hardhat network only"
```
?